lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -920,6 +922,7 @@ class MailboxDashBoardController extends ReloadableController
         accountId.value!,
         listEmailNeedMarkAsRead.listEmailIds,
         readActions,
+        listPresentationEmail.firstOrNull?.mailboxContain?.mailboxId,
```

Comment:
How about if user mark multiple as read in search?

Reply:
Handled in `SearchEmailController->_initWorkerListener()`